applications/luci-app-p910nd/root/etc/hotplug.d/usb/50_usb_printer_hotplug

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
```

Comment:
It will be a bit more to think about: the script would get passed the USB-ID e.g. "3f0/4117/100". the safest way to help the user get this that I can think of is parse dmesg output for usblp entries, and build a list of USB-IDs. Having a LuCI.usb class would go some way to make this easier. Thoughts?

Reply:
The USB ID is necessary to anchor to the correct device: need to be sure we don't send firmware to the wrong device.